server/utils/shabads/create-shabad-description.js

```diff
@@ -0,0 +1,16 @@
+import { translationMap, getWriter, getSource } from '../../../src/js/util/api/shabad';
+
+/**
+ *
+ * @param {object} shabad - The shabad obj
+ * @param {object} shabadInfo - The shabadInfo obj
+ **/
+export const createShabadDescription = (shabad, shabadInfo) => {
+
+  if (!shabad) return '';
+  const writerName = getWriter(shabad).english;
+  const shabadMeaning = translationMap.english(shabad);
```

Comment:
I think we should also add unicode in description.

Reply:
:+1: